README.md

```diff
@@ -17,9 +17,29 @@ i3spotifystatus is a tiny python (with even smaller bash script because I was to
 
 * clone repository to your prefered location
 * clone [@csssuf](https://github.com/csssuf)'s awk script to folder containing both `pystatus.py` and `getinfo.sh`
```

Comment:
Please remove this line, as now it's redundant

Reply:
I thought you would want the reference to the original file. I'll remove it then. :)